todo_app/flask_app/flask_app.py

```diff
@@ -0,0 +1,41 @@
+
+from flask.globals import request
+from flask import Flask, render_template, redirect
+from todo_app.trello.trello_api import TrelloAPI
+from todo_app.view_models.trello_view_model import TrelloViewModel 
+
+class FlaskApp:
```

Comment:
Not sure if I'm missing something, but I can't see anywhere this is actually used?